Example/Glia Sample Integration.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,455 @@
+				);
+				MARKETING_VERSION = 1.0;
+				PRODUCT_BUNDLE_IDENTIFIER = "Glia.Glia-Sample-Integration";
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_EMIT_LOC_STRINGS = YES;
+				SWIFT_VERSION = 5.0;
+				TARGETED_DEVICE_FAMILY = "1,2";
+			};
+			name = Debug;
+		};
+		C03A802D292B7A6700DDECA6 /* Release */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
+				CODE_SIGN_STYLE = Automatic;
+				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_ASSET_PATHS = "\"Glia Sample Integration/Preview Content\"";
+				DEVELOPMENT_TEAM = XR3G5NDPNH;
+				ENABLE_PREVIEWS = YES;
+				GENERATE_INFOPLIST_FILE = YES;
+				INFOPLIST_KEY_NSCameraUsageDescription = "";
+				INFOPLIST_KEY_NSMicrophoneUsageDescription = "";
+				INFOPLIST_KEY_NSPhotoLibraryAddUsageDescription = "";
+				INFOPLIST_KEY_UIApplicationSceneManifest_Generation = YES;
+				INFOPLIST_KEY_UIApplicationSupportsIndirectInputEvents = YES;
+				INFOPLIST_KEY_UILaunchScreen_Generation = YES;
+				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPad = "UIInterfaceOrientationPortrait UIInterfaceOrientationPortraitUpsideDown UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
+				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPhone = "UIInterfaceOrientationPortrait UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
+				IPHONEOS_DEPLOYMENT_TARGET = 16.1;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
+				MARKETING_VERSION = 1.0;
+				PRODUCT_BUNDLE_IDENTIFIER = "Glia.Glia-Sample-Integration";
+				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_EMIT_LOC_STRINGS = YES;
+				SWIFT_VERSION = 5.0;
+				TARGETED_DEVICE_FAMILY = "1,2";
+			};
+			name = Release;
+		};
+/* End XCBuildConfiguration section */
+
+/* Begin XCConfigurationList section */
+		C03A8018292B7A6700DDECA6 /* Build configuration list for PBXProject "Glia Sample Integration" */ = {
+			isa = XCConfigurationList;
+			buildConfigurations = (
+				C03A8029292B7A6700DDECA6 /* Debug */,
+				C03A802A292B7A6700DDECA6 /* Release */,
+			);
+			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
+		};
+		C03A802B292B7A6700DDECA6 /* Build configuration list for PBXNativeTarget "Glia Sample Integration" */ = {
+			isa = XCConfigurationList;
+			buildConfigurations = (
+				C03A802C292B7A6700DDECA6 /* Debug */,
+				C03A802D292B7A6700DDECA6 /* Release */,
+			);
+			defaultConfigurationIsVisible = 0;
+			defaultConfigurationName = Release;
+		};
+/* End XCConfigurationList section */
+
+/* Begin XCRemoteSwiftPackageReference section */
+		C03A8042292B895B00DDECA6 /* XCRemoteSwiftPackageReference "ios-sdk-widgets" */ = {
+			isa = XCRemoteSwiftPackageReference;
+			repositoryURL = "https://github.com/salemove/ios-sdk-widgets.git";
+			requirement = {
+				branch = master;
+				kind = branch;
```

Comment:
I'd like to propose using the latest released version but not a `master` branch because it is the guarantee that the example project will use the latest release, but not not-released changes. does it make sense to me guys? @gersonnoboa  @igorkravchenko

Reply:
Fixed [d992bda](https://github.com/salemove/ios-sdk-widgets/commit/d992bda8093d149fb48112026a37b81e7267537c)